src/ZODB/DemoStorage.test

```diff
@@ -375,12 +385,12 @@ Now, we create a demostorage.
 
 If we ask for an oid, we'll get 1042.
 
-    >>> u64(storage.new_oid())
+    >>> print(u64(storage.new_oid()))
```

Comment:
Because `u64` returns a long-integer, and one of the platforms (probably jython) returned `1042L` instead of `1042`, thus breaking the doctest. Using `print` causes `__str__` to be called instead of `__repr__` and gets consistent results everywhere.